.github/workflows/check-tags.yaml

```
@@ -0,0 +1,27 @@
+name: Check tag files
+
+on:
+  push:
+    branches: [ "master" ]
+  pull_request:
+    branches: [ "master" ]
+
+permissions:
+  contents: read
+
+jobs:
+  tags:
+    runs-on: ubuntu-latest
+
+    name: Check tag files
+    steps:
+      - uses: actions/checkout@v4
+      - name: Set up Ruby
+        uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: 3.2.4
+          bundler-cache: true
+      - name: Check tag files are properly updated
+        run: |
+          bundle exec rake update_tags
+          git diff --quiet
```

Great idea!!

```suggestion
          git diff --exit-code
```

How about using `--exit-code`?
We can check the diff in the CI result.

---

If there are large diffs, `git diff` may go into interactive mode.
I aim to this just prevent that.

---

I see!
Then this?

```suggestion
          git --no-pager diff --exit-code
```

---

I see, `--exit-code` is also good.

---

Fixed with https://github.com/fluent/fluentd-website/pull/379/commits/b9b03b16c8549164c4382bc42c04e6594ebe6292